### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by josedonizetti**
### 1. Explain what the PR does

Fix https://github.com/aquasecurity/tracee/issues/3167

1 - fix broken links
2 - extend events overview section to document default events, and sets
3 - add overview section to custom events
4 - add unexported extra events to mkdocs

### 2. Explain how to test it

```
make -f builder/Makefile.mkdocs
```

### CODE REVIEW

**Review by rafaeldtinoco — APPROVED** [review]
LGTM, feel free to wait for other reviews if you wish.

File: docs/docs/events/custom/overview.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Custom Events
+
+Tracee comes with lots of events, but you can extend it with events specific to your use case. There are two ways to extend Tracee with your own events:
+
+1. [Go](./golang.md)
+2. [Rego](./rego.md)
+
```
Comment (yanivagman) [inline docs/docs/events/custom/overview.md:7]: We should add something about `--signatures-dir` that can be used to point to a user defined directory containing signatures

**Review by yanivagman — CHANGES_REQUESTED** [review]
Missing documentation of `--signatures-dir` in the custom events overview.

In addition the getting_started/overview page needs some fixes:
In these two sentences the links are broken:
```
In the [Docs](https://aquasecurity.github.io/tracee/v0.14/docs) section you can find the complete reference documentation for all of the different features and settings that Tracee has to offer.
In the [Contributing](https://aquasecurity.github.io/tracee/v0.14/contributing) section you can find technical developer documentation and contribution guidelines.
```

Plus we need to remove this sentence in the same getting started page:
`To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee/).`

File: docs/docs/events/custom/overview.md (hunk 1 above)
Comment (josedonizetti) [reply docs/docs/events/custom/overview.md:7]: But this is done in `rego`already, why do it again? https://aquasecurity.github.io/tracee/v0.14/docs/events/custom/rego/

**Comment by josedonizetti** [comment]
> Missing documentation of `--signatures-dir` in the custom events overview.
> 
> In addition the getting_started/overview page needs some fixes: In these two sentences the links are broken:
> 
> ```
> In the [Docs](https://aquasecurity.github.io/tracee/v0.14/docs) section you can find the complete reference documentation for all of the different features and settings that Tracee has to offer.
> In the [Contributing](https://aquasecurity.github.io/tracee/v0.14/contributing) section you can find technical developer documentation and contribution guidelines.
> ```

But they are fixed in the PR already -> https://github.com/aquasecurity/tracee/pull/3175/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R16-R17 -> aren't the fixes working for you?

**Comment by yanivagman** [comment]
> > Missing documentation of `--signatures-dir` in the custom events overview.
> > In addition the getting_started/overview page needs some fixes: In these two sentences the links are broken:
> > ```
> > In the [Docs](https://aquasecurity.github.io/tracee/v0.14/docs) section you can find the complete reference documentation for all of the different features and settings that Tracee has to offer.
> > In the [Contributing](https://aquasecurity.github.io/tracee/v0.14/contributing) section you can find technical developer documentation and contribution guidelines.
> > ```
> 
> But they are fixed in the PR already -> https://github.com/aquasecurity/tracee/pull/3175/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R16-R17 -> aren't the fixes working for you?

Sorry, missed that

**Review by yanivagman — APPROVED** [review]
Ok, so signatures-dir exist for rego, but maybe we want to add a few more words about it in the overview?
Anyway, LGTM

**Pull request merged**